File: EaterSpring/src/main/kotlin/xyz/purema/eater/dtos/form/LoginForm.kt
package xyz.purema.eater.dtos.form

import org.hibernate.validator.constraints.Length
import javax.validation.constraints.NotEmpty

class LoginForm {
    @field:NotEmpty(message = "Please enter user id")
    var userId: String? = null

    @field:NotEmpty(message = "Please enter password")
    @field:Length(min = 6, message = "Password must have minimum of 6 characters")
    var password: String? = null
}

File: EaterNet/Eater/Dtos/Form/LoginForm.cs
using System.ComponentModel.DataAnnotations;

namespace Eater.Dtos.Form
{
    public class LoginForm
    {
        [Required(ErrorMessage = "Please enter user id")]
        public string UserId { get; set; }

        [Required(ErrorMessage = "Please enter password")]
        [MinLength(6, ErrorMessage = "Password must have minimum of 6 characters")]
        public string Password { get; set; }

        public LoginForm(string userId, string password)
        {
            UserId = userId;
            Password = password;
        }
    }
}
